setup.py

```diff
@@ -26,6 +26,8 @@
     },
     packages=find_packages(exclude=["contrib", "docs", "tests"]),
     install_requires=[
+        "click",
+        "dcm2bids==2.1.4",
```

Comment:
You can be a bit more flexible:

```suggestion
        "dcm2bids~=2.1.4",
```